tiatoolbox/utils/transforms.py

```diff
@@ -119,16 +119,48 @@ def imresize(img, scale_factor=None, output_size=None, interpolation="optimise")
         else:
             interpolation = "area"
 
-    if img.dtype == np.float16:
-        img = img.astype(np.float32)
+    # dtypes that cv2.resize can work on out-of-the-box
+    dtype_mapping = [
+        (np.bool, np.uint8),
+        (np.int8, np.int16),
```

Comment:
why int8 to int16?

Reply:
Updated description in https://github.com/TissueImageAnalytics/tiatoolbox/pull/328/commits/5b68415f24b1aeda108f5473f66f8a1f81d0bfab